### ISSUE TITLE
Website Responsiveness Enhancement

### ISSUE THREAD

**Issue by CharanAdithya6505** [arghadipmanna101/Flipkart_Clone#11]
Our open-source organization's website lacks responsiveness across various devices and screen sizes, leading to suboptimal user experience. Improving responsiveness is crucial for ensuring accessibility and usability for all users.Please assign this issue to me, i would love to resolve this issue

**Comment by arghadipmanna101** [comment]
ok go ahead.

**Comment by arghadipmanna101** [comment]
@CharanAdithya6505  ok go ahead.

**Comment by Shyambarua** [comment]
Assign this issue to me 
As a full stack Developer i can write optimized code and responsive and i can integrate with backend also.

**Comment by sid346184** [comment]
Before

https://github.com/arghadipmanna101/Flipkart_Clone/assets/71984091/1a8e84f7-1f49-48ec-947c-2e6d3045ecd6

After

https://github.com/arghadipmanna101/Flipkart_Clone/assets/71984091/add8b665-d7a7-49a7-8631-6aba79717175



These changes are made by me under #677 . Please merge it with gssoc label and yeah if you wish for more such contributions you can also assign me this under gssoc :)

**Comment by sid346184** [comment]
Hey, as a full stack web developer I would like to make the website more responsive and add some more features. 
Please assign me this under GSSOC
